### PR TITLE
fix: AU-796: Fix subvention validations.

### PR DESCRIPTION
**Pull request by jiisuominen**
# [AU-796](https://helsinkisolutionoffice.atlassian.net/browse/AU-796)

There was error in subvention validations, where you had to have ALL fields in subventions filled or form errored. Desired functionality is that ONE of the fields is required, no matter what that is.

## What was done

* Added validation for subvention fields that check empty or 0,00€ values

## How to install

* Make sure your instance is up and running on correct branch.
  * `git checkout UHF-0000_insert_correct_branch`
  * `make fresh`
* Run `make drush-cr`

## How to test

* [x] Go to KASKO form, create a new one.
* [x] Leave subvention values as defaults and try to go to next page. Should see error.
* [ ] Same thing with values inserted and then removed, should fail.
* [ ] And the code standards.

[AU-796]: https://helsinkisolutionoffice.atlassian.net/browse/AU-796?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by strangeworlder** [comment]
Näyttökuva 2023-3-16 kello 17 26 39
Tulee erroria lomakkeella

**Comment by strangeworlder** [comment]
Näyttökuva 2023-3-16 kello 17 26 39

Kaskon lomakkeen toisella sivulla, tulee tällainen errorlista

File: public/modules/custom/grants_handler/src/Element/CompensationsComposite.php [hunk 1]
```diff
@@ -44,20 +41,53 @@ public static function getCompositeElements(array $element): array {
...
+    foreach ($values['subventions'] as $item) {
+      if(isset($item['amount'])){
+        if ($item['amount'] == '0,00€' || empty($item['amount'])) {
+          $zeroes++;
+        }
+      }
+    }
+
+    if ($zeroes === $subventionNumber) {
+      $formState->setErrorByName('subventions', t('You must insert atleast one subvention amount'));
```
Comment (strangeworlder) [inline public/modules/custom/grants_handler/src/Element/CompensationsComposite.php:81]: "at least"
Missing fi and sv translations

**Review by strangeworlder — CHANGES_REQUESTED** [review]

File: public/modules/custom/grants_handler/src/Element/CompensationsComposite.php (hunk 1 above)
Comment (jiisuominen) [reply public/modules/custom/grants_handler/src/Element/CompensationsComposite.php]: Done.

**Review by strangeworlder — APPROVED** [review]

**Review by strangeworlder — APPROVED** [review]

**Pull request merged**